### PR TITLE
[fix](olap) deletion statement with space conditions did not take effect

### PR DESCRIPTION
**Pull request by mrhhsg**
## Proposed changes

Deletion statement like this:
```sql
delete from tb where k1 = '  ';
```

The rows whose `k1`'s value is ' ' will not be deleted.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

File: be/src/olap/delete_handler.cpp [hunk 1]
```diff
@@ -110,7 +110,7 @@ std::string DeleteHandler::construct_sub_predicates(const TCondition& condition)
         } else if (op == "!*=") {
             op = "!=";
         }
-        condition_str = condition.column_name + op + condition.condition_values[0];
+        condition_str = condition.column_name + op + "'" + condition.condition_values[0] + "'";
```
Comment (yiguolei) [inline be/src/olap/delete_handler.cpp:113]: If we upgrade from doris 1.2 and there is a delete condition. It does not contains '', is there error?

Comment (yiguolei) [reply be/src/olap/delete_handler.cpp:113]: because it already save the previous condition does not have '' in data file.

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**